pkg/charts/linkerd2/tracing.go

```diff
@@ -0,0 +1,60 @@
+package linkerd2
+
+import (
+	"k8s.io/helm/pkg/chartutil"
+)
+
+type (
+
+	// Tracing consists of the add-on configuration of the distributed tracing components sub-chart.
+	Tracing struct {
+		Enabled   bool       `json:"enabled"`
+		Collector *Collector `json:"collector"`
+		Jaeger    *Jaeger    `json:"jaeger"`
+	}
+
+	// Collector consists of the config values required for Trace collector
+	Collector struct {
+		Name      string     `json:"name"`
+		Image     string     `json:"image"`
+		Resources *Resources `json:"resources"`
+	}
+
+	// Jaeger consists of the config values required for Jaeger
+	Jaeger struct {
+		Name      string     `json:"name"`
+		Image     string     `json:"image"`
+		Resources *Resources `json:"resources"`
+	}
+)
+
+var (
+	tracingChartName   = "tracing"
+	tracingConfigStage = []string{
+		"templates/tracing-rbac.yaml",
+	}
+
+	tracingControlPlaneStage = []string{
+		"templates/tracing.yaml",
+	}
+)
+
+// GetChartName returns the name of the add-on sub-chart
```

Comment:
All of these look like static values that can be read from the `tracing` subtree in `values.yaml`.